libs/community/langchain_community/tools/mindsdb/tool.py

```diff
@@ -0,0 +1,20 @@
+from typing import Text, Optional
+
+from langchain_community.utilities.mindsdb import BaseMindWrapper
+from langchain_core.callbacks import CallbackManagerForToolRun
+from langchain_core.tools import BaseTool
+
+
+class AIMindTool(BaseTool):
+    name: Text = "ai_mind"
+    description: Text = (
+
```

Comment:
missing a description

Reply:
Hey @baskaryan,
Thank you for taking the time to review this PR. I have added a description and also resolved the lint issues as best I can. Please let me know if you have any other comments.